07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<p>Does the <code>autoload</code> extension work if you run it <em>after</em> importing your modules? I had in mind that it had to be run before, but I might be wrong.</p><p> </p><p>You don't have to change anything, this is just a question I have.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>It does seem to work where I have it here. Not sure exactly how it does it, but I haven't had any issues.</p>